api/docs/scatter_gather_emulation.dox

```diff
@@ -61,7 +61,7 @@ be complete.
 
 
 ```
-vpscatterdd {%k1} %xmm10 -> %rax(,%xmm11,4)[4byte] %k1
+{%raw %}vpscatterdd {%k1} %xmm10 -> %rax(,%xmm11,4)[4byte] %k1{%endraw %}
```

Comment:
For the direct (not for embedding in Jekyll: `api/docs/html` rather than `api/docs/embed/html`) html output for this doc, what does this look like?  I assume it has this `{% raw %}` there.  Would an alternative solution of `&lt;%k1}` solve the Jekyll issue while also looking fine in the direct html output?

Reply:
> what does this look like? I assume it has this {% raw %} there.

It shows the `{%raw %}` and `{%endraw %}` as is. I saw this before but assumed it's okay as the website contains the Jekyll one only. And we had an existing use of this workaround: https://github.com/DynamoRIO/dynamorio/blob/c99bcafc7e9057e0adb83fc42784a2fb1220e27e/clients/drdisas/drdisas.dox#L45

> Would an alternative solution of &lt;%k1} solve the Jekyll issue while also looking fine in the direct html output?

I can try.

Reply:
Whatever choice we make let's make that drdisas one do the same thing as the ones here.

Reply:
> Would an alternative solution of &lt;%k1} solve the Jekyll issue while also looking fine in the direct html output?

It didn't. Perhaps because this is in a code block, the '&' itself gets converted to &amp. So the html output shows `&gt;`.

Also, I assume you meant `&#123;` for `{`.

Reply:
Do we use the `api/docs/html` output for anything? Or is it okay for it to have the `{% raw %}` tag.

Reply:
Yes, that output is placed into the release package.  If you untar/unzip any of our release packages it is right there for local browsing.  If we had a Windows installer (instead of just zip file) like Dr. Memory does it would create a "Start Menu" link that points to that html output.

Reply:
Maybe the best solution is a regex replacement that auto-adds `{% raw %}...{% endraw %}` around `{%.*}` in the existing cmake code that creates the embedded docs.  Then the regular html is fine and nobody will add future disassembly and hit this same problem like may well happen if we rely on a manual fix each time.

Reply:
Done. Instead of adding `{% raw %}...{% endraw %}` around `{%.*}`, I added changes to add `{% raw %}...{% endraw %}` around `{%` and `%}` separately. I think this is better because `{%` and `%}` both can occur independently (here we have only the former though). Also, `{` and `}` alone are not problematic afaik.

Reply:
So this approach also assumes that we won't have any actual use of Jekyll tags in the file. I suppose that's okay?

Reply:
I think so, yes.